CHANGES.rst

```diff
@@ -226,6 +226,8 @@ Bug Fixes
   - Fix use of the ``relax`` keyword in ``to_header`` when used to change the
     output precision. [#5164]
 
+  - Fix rotateCD(). [#5189]
```

Comment:
@astrofrog Will do.
I think we are leaning towards a consensus to eventually remove this method. Any objections?